### ISSUE TITLE
Variance in Global Total Cases and Death between App & Web?

### ISSUE THREAD

**Issue by travelkev** [WorldHealthOrganization/app#1692]
## Summary

Is there a variance between the global total cases and deaths as reported on https://covid19.who.int/ and in the app?

## Reproduction Steps

1. Installed V0.13.0 (24) app via TestFlight
2. On home page click on "Global/National Cases" bar at the start of the page to navigate to the "Recent Numbers" page.
3. Scroll down to Global figures and ensure that the page is set to show "Total" not "Daily".
4. When testing at 13h00 on 28th Oct: -

MyHealthApp showed - 32,578,382 Total Cases and 860,401 Total Deaths
https://covid19.who.int/ showed - 43,540,739 total confirmed cases and 1,160,650 deaths

## Expected Behavior

National figures agree between app and website and so I was expecting these total figures to also agree, mindful that current ay reporting is often incomplete.

## System

- Device: iPhone7
- App Version: 0.13.0 (24)

## Screenshots

<details>
<summary>Screenshots</summary>
Screenshot Web
![28OctFiguresWeb](https://user-images.githubusercontent.com/12529368/97433217-7275c980-191d-11eb-9b30-6c669b123798.JPG)

Screenshot Mobile App
![28OctFiguresApp](https://user-images.githubusercontent.com/12529368/97433374-b072ed80-191d-11eb-9c6b-1bd47b197d48.JPG)

</details>

## Checklist:

- [X ] Searched the [existing issues](https://github.com/WorldHealthOrganization/app/issues) to ensure you are not creating a duplicate.
- [ X] Followed the [Contributor Guidelines](https://github.com/WorldHealthOrganization/app/blob/master/docs/CONTRIBUTING.md).

**Comment by Mohit020888** [comment]
I am also able to reproduce the bug on my Huawei Honor 7c running on Android 8.0 , 
App Version : [0.14.0 + 25]

**Comment by theswerd** [comment]
@travelkev so it displayed your first screenshot

**Comment by brunobowden** [comment]
@travelkev - this sounds like caching old data but I'm suspicious that @Mohit020888 is seeing the same thing. I've got 0.13.0 (24) and it's displaying correctly for me.

@travelkev @Mohit020888 - please try a few things:
1) Make sure you've got internet connection on phone, return to app after 20 min and see if the numbers update (it should refresh every 10min when returning to app).
2) Close app and restart it. Check numbers again.
3) Uninstall app and install it again. Check numbers again.

Let me know what you learn from this.

**Comment by Mohit020888** [comment]
@brunobowden - I doubt the app data is getting refreshed after 20 minutes however the app data is in sync with the official WHO dashboard now post midnight (IST) 
![Screenshot_20201029-003201](https://user-images.githubusercontent.com/63721227/97484235-800f6b80-197e-11eb-927e-62d88680a170.jpg)

**Comment by brunobowden** [comment]
@travelkev - is this now showing correctly for you?

**Comment by Mohit020888** [comment]
@brunobowden @theswerd - the app is again showing incorrect and outdated Global cases and deaths.
![Screenshot_20201030-161932](https://user-images.githubusercontent.com/63721227/97696751-071e2a00-1acc-11eb-8c4b-65dcaa1e5ab0.jpg)

**Comment by brunobowden** [comment]
This could also be a failure in fetching the content when reopening the app.

@theswerd - it would be good to have a warning mechanism for old content:
- Alert: "Content is XX hours old / YY days old" (e.g. [main.dart Alert](https://github.com/WorldHealthOrganization/app/blob/master/client/lib/main.dart#L210-L212))
- Display only if:
   - Last update more than 4 hours old
   - Attempt to update content upon app open has timed out

**Comment by theswerd** [comment]
Confirming we want this to look the same as the development warning?

**Comment by brunobowden** [comment]
@theswerd - probably a less confrontational alert. Since this is for the general public, we also need to be more mindful of accessibility. So maybe black text on a pale blue background (it should be distinct from the dark blue background of the "Protect Yourself" section on the main page.

**Comment by brunobowden** [comment]
@travelkev @Mohit020888 - we can't reproduce this issue though I believe you both saw it. Please comment here if it happens again as it is an important issue but not something we have a good lead on. See updated comments in description for details.

**Comment by brunobowden** [comment]
@theswerd - let's de-prioritize this to work on other issues

**Comment by Mohit020888** [comment]
@brunobowden - Now I am seeing daily cases count as 0 for US on the stats screen.
I have uninstalled and reinstalled the app but the issue still exists.

![Screenshot_20201109-134445](https://user-images.githubusercontent.com/63721227/98516886-1b63e300-2293-11eb-8827-f301b0fdbfb9.jpg)

**Comment by brunobowden** [comment]
Good catch. The data is wrong in the [WHO COVID-19 dashboard](https://covid19.who.int/table). It shows "0" cases for Nov 8th for at least 3 countries. @theswerd and I were talking about this earlier today: https://who-app.slack.com/archives/C018Y9Q07PX/p1604873092047200. I've emailed WHO to find out what we can do about it.

@Mohit020888 - please create a separate bug about this. I'm still pondering what's the best way to fix it. Some countries do legitimately have zero cases.... so we need to figure out when it's legitimately zero or an error.

**Comment by Mohit020888** [comment]
@brunobowden - I have created https://github.com/WorldHealthOrganization/app/issues/1724 for the above issue and marked it as P2.

**Comment by brunobowden** [comment]
Deferring to next release.... though we'll monitor if issues occur with this one.

**Comment by Mohit020888** [comment]
tested on Release Candidate Version 1.0.0-staging(281) , the app showed stale data for Global Cases till yesterday night i.e. 20th December 2020 and showed only 49,720,550 Global Cases but now past midnight the app is showing correct number of Global cases 75,110,651.
<details>
![wrong_global_stats](https://user-images.githubusercontent.com/63721227/102723115-f6e83580-432b-11eb-86a7-e5c3c0c3e949.jpg)
![correct_global_cases](https://user-images.githubusercontent.com/63721227/102723118-f8b1f900-432b-11eb-9af7-8e7b50052bdd.jpg)

</details>

**Comment by theswerd** [comment]
I just ran the app on my simulator and got this:

<img src="https://user-images.githubusercontent.com/38309438/103107441-29778400-45f3-11eb-95b6-ed257af1481b.png" height="400px"></img>

This makes me think it is **not a cache bug** as this was a fresh simulator
